### PR TITLE
dev-python/flask-bootstrap: new package

### PR DESCRIPTION
**Pull request by sbraz**
It had a lot of deps, especially for tests. I decided to add them as they might be useful.
Reviews are welcome, I might have made some mistakes since there is a lot of packages here.

### CODE REVIEW

**Comment by monsieurp** [comment]
@gentoo/python @gentoo/proxy-maint

**Comment by rindeal** [comment]
instead of overwriting `S`, you can `inherit vcs-snapshot`

**Comment by sbraz** [comment]
Oh thanks, I completely forgot about it :)

**Comment by sbraz** [comment]
@rindeal done. You meant for dominate only, right? Not for flask-debug?

**Comment by rindeal** [comment]
I meant all occurrences as I see no reason why not. For flask-debug you could then also remove `MY_PN`

**Comment by sbraz** [comment]
Feels a bit weird to use an eclass for VCS snapshots just because the tarball isn't named properly, I'd like more feedback before I do it.

**Comment by rindeal** [comment]
git tag is just a human-friendly name for a commit hash, so if you used vcs-snapshot.eclass for a commit-based archive, then, based on the same logic, you can use it for tags as well.

**Comment by sbraz** [comment]
But it's not about git… Even if I downloaded the package from PyPI, it would be named Flask-Debug.

**Comment by rindeal** [comment]
Ok, to make it clear, as long as your `SRC_URI` contains pattern `.*github.com.*/archive/.*`, you can use `vcs-snapshot` right away, otherwise only after hesitation.

**Comment by sbraz** [comment]
But I'm just using it because PyPI does not have tests. Once upstream fixes this, I don't plan on using github any more. So I think it would be wrong to use vcs-snapshot here.

**Comment by rindeal** [comment]
Do as you wish. And why do you prefer PyPI?

**Comment by sbraz** [comment]
Well PyPI is the standard home of Python packages, all ebuilds do that :)

File: dev-python/flask-nav/flask-nav-0.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Easily create navigation for Flask applications."
+HOMEPAGE="http://pythonhosted.org/flask-nav/"
```
Comment (gktrk) [inline dev-python/flask-nav/flask-nav-0.5.ebuild]: https://pythonhosted.org/flask-nav/
      ^ prefer https when available? ;)

**Comment by gktrk** [comment]
The ebuilds look cool. I got stuck on the vcs-snapshot for a moment too, but it's either this or switch to pypi and RESTRICT tests until upstream fixes it. I didn't download the sources to double check the deps or the licenses though, and didn't give it a runtest. Way too busy this week :( If this PR is still around next week, I'll give it a runtest.

**Comment by idella** [comment]
pypi is the preferred source location to acquire tarballs, however the tarballs uploaded there are often missing content. Acquisition from gtihub is often done in order to acquire a proper quality tarball. They are notorious for missing adding testsuites.  Managing alternate tarballs in source repos is a normal practice in pythonic packages. 
Be careful of adhering strictly to one work flow here.  Flexibility and compromise is quite a normal necessity.

This pull req will not get past tomorrow so I would not be concerned.

**Comment by idella** [comment]
There is alot to do here which is why I have put it on hold 'til first thing in the morning. From top down:

1. dominate-2.2.0.ebuild  I agree with acquiring the snapshot yielding
SRC_URI="https://github.com/Knio/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
It is commented with the explanation which is consistent with my teachings to Louis.  
Finally someone else dedicated to having testsuites honoured.
TICK since I cannot figure how to do a thumb up emoticon.
2.  RDEPEND=""  can be deleted,  no fatal, no prob.

3. dev-python/visitor is fine.
4. dev-python/inflection is fine.
5.  We have finally hit one of
```
loading intersphinx inventory from http://flask.pocoo.org/docs/objects.inv...
loading intersphinx inventory from https://docs.python.org/objects.inv...
```
in the doc build. I shall explain how to deal with this nasty in IRC.
do
`dev-python/flask-nav $ grep -r sphinx.ext.intersphinx ../`

6. ditto flask-appconfig
```
loading intersphinx inventory from http://docs.python.org/objects.inv...
loading intersphinx inventory from http://pythonhosted.org/Flask-SQLAlchemy/objects.inv...
WARNING: intersphinx inventory 'http://pythonhosted.org/Flask-SQLAlchemy/objects.inv' not fetchable due to <class 'urllib.error.HTTPError'>: HTTP Error 404: Not Found
loading intersphinx inventory from http://flask.pocoo.org/docs/objects.inv...

```
7. ditto dev-python/flask-debug. These are from the same author so he has set the same pattern in all flask-fleet. You will see I have dealt with the others in the past before you got here.

8. As expected ditto flask-bootstrap

**Pull request closed without merging**